src/graphics.c

```diff
@@ -8,6 +8,7 @@ See the file LICENSE for details.
 #include "kerneltypes.h"
 #include "font.h"
 #include "kernelcore.h"
+#include "math.h"
```

Comment:
also should I just add abs in this pr, do another pr for it, or wait for you to merge your math.h pr. I can always solve merge conflicts with abs